emmet-builders/requirements.txt

```diff
@@ -1,3 +1,3 @@
-pymatgen==2020.4.29
-maggma==0.20.0
+pymatgen>=2020.12.18
+maggma>=0.20.0
```

Comment:
requirements should be pinned to a version so github and dependabot can incrementally check it. If you need a newer version pin it to the version you need.

Reply:
OK